pkg/config/application.go

```diff
@@ -143,16 +143,16 @@ func RemoveAnchorPeer(config *cb.Config, orgName string, anchorPeerToRemove *Anc
 	return fmt.Errorf("could not find anchor peer %s:%d in %s's anchor peer endpoints", anchorPeerToRemove.Host, anchorPeerToRemove.Port, orgName)
 }
 
-// GetAnchorPeer retrieves existing anchor peers from a application organization.
-func GetAnchorPeer(config *cb.Config, orgName string) ([]*AnchorPeer, error) {
+// GetAnchorPeers retrieves existing anchor peers from a application organization.
+func GetAnchorPeers(config cb.Config, orgName string) ([]AnchorPeer, error) {
```

Comment:
Given all the other functions take a pointer, this will be inconsistent and require a dereference.